.github/workflows/3-last-step.yml

```diff
@@ -24,78 +24,18 @@ jobs:
     name: Find Exercise Issue
     uses: skills/exercise-toolkit/.github/workflows/find-exercise-issue.yml@v0.6.0
 
-  # This job is optional. We often call it a "grading job". If the step is not graded, remove it.
-  check_step_work:
```

Comment:
Removed to make the exercise template simpler. This job is optional and can be added to the last (or any) step if needed.